misk/api/misk.api

```diff
@@ -777,60 +777,6 @@ public abstract class misk/concurrent/WrappingListeningExecutorService : com/goo
 	protected abstract fun wrap (Ljava/util/concurrent/Callable;)Ljava/util/concurrent/Callable;
 }
 
-public abstract interface annotation class misk/config/AppName : java/lang/annotation/Annotation {
```

Comment:
Alternatively, if you've tried building a misk service with these changes and it didn't require any changes, that will make me much happier!

Reply:
I chatted further with @tso and he also helpfully pointed this out. I might leave a copy but with deprecation annotation in the old location for now. This would let this PR land and rollout and then say a month from now, the duplicate is removed once services have migrated. Similar to the approach in https://github.com/cashapp/misk/pull/2731 

Does that sound feasible?

Reply:
Yep!